test/extended/testdata/imagestream-jenkins-slave-pods.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  labels:
+    role: jenkins-slave
```

Comment:
not required ... uses the image stream name if not slave-label field set ... we set it with the imagestream tag, aside from testing,  since a) its name will never be just "jenkins-slave" since the tag name is appended b) we can then reuse the same bc for all three configmap/imagestream/imagestreamtag